src/main/scala/firrtl/stage/phases/CatchExceptions.scala

```diff
@@ -32,10 +35,27 @@ class CatchExceptions(val underlying: Phase) extends Phase {
     /* Rethrow the exceptions which are expected or due to the runtime environment (out of memory, stack overflow, etc.).
      * Any UNEXPECTED exceptions should be treated as internal errors. */
     case p @ (_: ControlThrowable | _: FIRRTLException | _: OptionsException | _: FirrtlUserException |
-        _: FirrtlInternalException | _: PhaseException | _: DependencyManagerException) =>
-      throw p
-    case CustomTransformException(cause) => throw cause
-    case e: Exception => Utils.throwInternalError(exception = Some(e))
+        _: FirrtlInternalException | _: PhaseException | _: DependencyManagerException | _: AssertionError) => {
+      dumpOnError(underlying, a)
+      throw p }
+    case CustomTransformException(cause) => {
+      dumpOnError(underlying, a)
+      throw cause }
```

Comment:
I think you're going to need to run Scalafmt: `sbt scalafmtAll`

Reply:
Also note that it's not necessary to wrap the right-hand side of a `case` in curly braces, and I guess stylistically discouraged (by me)

Reply:
will `sbt scalafmtAll` before next push